core/.changelog.d/4351.changed.1

```diff
@@ -0,0 +1 @@
+Forbidden multisig to singlesig change outputs.
```

Comment:
I think the message should read "Forbid ..." or possibly "Forbade ...".

Reply:
Done in https://github.com/trezor/trezor-firmware/pull/4351/commits/54e4e27910b05c33658dfe5dd2f7d4ed7448c97d.